node/src/components/network/tests_bulk_gossip.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(unreachable_code)] // TODO: Figure out why this warning triggers.
```

Comment:
Maybe a forgotten `#[cfg(test)]` on this module?

Reply:
Unfortunately no :(

```
#[cfg(test)]
mod tests_bulk_gossip;
```

in `network.rs`.